superqt/combobox/_test/enum_comb_box.py

```diff
@@ -0,0 +1,102 @@
+from enum import Enum
```

Comment:
could you rename this file `test_enum_comb_box.py`, and rename the directory `_tests` (plural) to match the rest of the repo?